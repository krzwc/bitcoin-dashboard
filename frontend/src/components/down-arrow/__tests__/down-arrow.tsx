import React from 'react';
import DownArrow from 'components/down-arrow';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('DownArrow', () => {
    test('snapshot test', () => {
        const wrapper = shallow(<DownArrow className="css-class" />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
