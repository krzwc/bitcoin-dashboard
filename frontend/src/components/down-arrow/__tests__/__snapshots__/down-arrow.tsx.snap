// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DownArrow snapshot test 1`] = `
<div
  className="down-arrow css-class"
>
  <span
    dangerouslySetInnerHTML={
      Object {
        "__html": "test-file-stub",
      }
    }
  />
</div>
`;
